pages/casestudies/index.js

```diff
@@ -0,0 +1,49 @@
+import GenericLayout from "../../components/layout/GenericLayout";
+import CaseStudyCard from '../../components/CaseStudyCard';
+import Paragraph from '../../components/typography/Paragraph';
+import TextLink from '../../components/typography/TextLink';
+import Heading from "../../components/typography/Heading";
+import CaseStudiesList from "../../config/case-studies.json";
+
+export default function casestudies() {
+  const description =
+    "Learn about different case studies based on AsyncAPI spec and related tools.";
+  const image = "/img/social/meetings.png";
+  const title = "Case Studies";
+
+  return (
+    <GenericLayout
+      title={title}
+      description={description}
+      image={image}
+      wide
+    >
+
+      <div className="py-10 relative max-w-xl mx-auto px-4 sm:px-6 lg:px-8 lg:max-w-screen-xl">
+        <div className="grid lg:grid-cols-9 lg:gap-8 lg:text-center">
+          <div className="col-start-3 col-span-5">       
```

Comment:
Why are we using grid CSS here? After inspecting all the elements, I found no use for why the grid has been used here.

Reply:
I wish I new 😆 
can you advice which part you think I should remove?

Reply:
Can I directly commit to this PR regarding the UI changes? There will be slight changes only related to CSS classes.

Reply:
oh yeah man, be my guest ❤️